scoring_engine/checks/bin/telnet_check

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# A scoring engine check that logs into telnet and runs a command
+#
+# To install: pip install -I "telnetlib3==1.0.1"
+
+import sys
+import telnetlib
+
+if len(sys.argv) != 7:
+    print("Usage: " + sys.argv[0] + " host port username password commands")
+    print("commands parameter supports multiple commands, use ';' as the delimeter")
+    sys.exit(1)
+
+host = sys.argv[1]
+port = sys.argv[2]
+timeout = sys.argv[3]
+username = sys.argv[4]
+password = sys.argv[5]
+commands = sys.argv[6].split(';')
+
+
+def connect_and_execute(host, port, timeout, username, password, command):
```

Comment:
I don't think this actually executes any commands, looks like the "command" variable name is never used in this function.